### PR TITLE
Dig 19 create start test page

### PR DESCRIPTION
**Pull request by k-henningson**

### CODE REVIEW

**Comment by k-henningson** [comment]
Screen Shot 2022-11-30 at 11 05 16 AM

File: modules/components/TestBox.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    return (
+        <>
+            <Pressable
+                onPress={() => setShowModal(true)}
+                width="50%"
+                rounded="lg"
+                overflow="hidden"
+                p="12"
+                my="3"
+                borderColor="coolGray.300"
```
Comment (j-dowell) [inline modules/components/TestBox.js:17]: Fine for a placeholder but we'll eventually need to make sure every component is themed properly

File: modules/components/TestBox.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                <Modal.Content maxWidth="350">
+                    <Modal.CloseButton />
+                    <Modal.Header>{props.children}</Modal.Header>
+                    <Modal.Body>
+                        <VStack space={3}>
+                            <HStack
+                                alignItems="center"
+                                justifyContent="space-between"
+                            >
+                                <Pressable
```
Comment (j-dowell) [inline modules/components/TestBox.js:41]: Extract the copy into a list and map over it, so you don't have to copy/paste the `<Pressable` component multiple times. It's more code to read, and if we ever need to change part of it, we need to change it in 5+ places.

e.g.
```
const tests = [
  { header: 'CTV', subHeader: '0 taps: Very Easy' },
...
]


// Then in the component:
...
{
  tests.map(test => <Pressable ...
}

```

File: modules/screens/test/TestPage.js [hunk 3]
```diff
@@ -1,12 +1,51 @@
...
+            <View
+                style={{
+                    flex: 1,
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                }}
+            >
+                <HStack>
+                    <TestBox>
+                        <StyledText>Compression Test (CT)</StyledText>
```
Comment (j-dowell) [inline modules/screens/test/TestPage.js:18]: Same kind of comment about hardcoding markup. Would be better to extract these into a list and map over it in the component

File: modules/components/TestBox.js (hunk 2 above)
Comment (k-henningson) [reply modules/components/TestBox.js]: Screen Shot 2022-12-02 at 3 29 19 PM

**Review by j-dowell — APPROVED** [review]

**Pull request merged**